plugins/lookup/filetree.py

```diff
@@ -45,46 +45,50 @@
     mode: '{{ item.mode }}'
   with_filetree: web/
   when: item.state == 'link'
+
+- name: list all files under web/
+  ansible.builtin.debug:
+    msg: "{{ lookup('filetree', 'web/') }}"
```

Comment:
```suggestion
    msg: "{{ lookup('community.general.filetree', 'web/') }}"
```
(The others are changed in #722.)